app/src/androidTest/java/net/skyscanner/backpack/toast/BpkToastTests.kt

```diff
@@ -37,13 +38,13 @@ class BpkToastTests : BpkSnapshotTest() {
   fun screenshotTestToast_Default() {
     val toast = BpkToast.makeText(testContext, "Test", BpkToast.LENGTH_SHORT)
     toast.show()
-    snap(toast.view)
+    snap(toast.view!!)
```

Comment:
With compileSdk=30 Toast.view was deprecated. The docs states it was done to stop the developers to put custom views in toasts.

However, we don't use it in this manner. We only need it to take a photo of Toast for tests ,ad it's working fine.